src/quo/components/wallet/network_amount/view.cljs

```diff
@@ -5,7 +5,18 @@
     [quo.components.utilities.token.view :as token]
     [quo.components.wallet.network-amount.style :as style]
     [quo.theme :as quo.theme]
-    [react-native.core :as rn]))
+    [react-native.core :as rn]
+    [schema.core :as schema]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:amount {:optional true} [:maybe :string]]
+      [:token {:optional true} [:or keyword? string?]]
```

Comment:
We haven't reached a conclusion yet about using keywords or functions as predicates, but here it would be better to be consistent and use only keywords or only functions.